src/styles/blocks/card.scss

```
@@ -0,0 +1,86 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid var(--card-border-color);
+  border-radius: 5px;
+  background-color: var(--light-color);
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+  }
+
+  &__product-name {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: var(--main-color);
+  }
+
+  &__product-info {
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: var(--secondary-color);
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review-text {
+    font-size: 10px;
+    line-height: 14px;
+    color: var(--main-color);
+  }
+
+  &__price-block {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    font-size: 12px;
+    line-height: 18px;
+    color: var(--secondary-color);
+  }
+
+  &__price {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    color: var(--main-color);
+  }
+
+  &__btn {
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    padding: 12px 70px;
+    background: var(--accent-color);
+    color: var(--light-color);
+    border-radius: 5px;
+    font-size: 14px;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-decoration: none;
+    max-width: 100%;
+    min-height: 40px;
```

check bug. when hover on button in top row, bottom row moving.

```suggestion
    width: 100%;
    border: 1px solid var(--accent-color);
    line-height: 16px;
```